controller/api/util/api_utils.go

```diff
@@ -535,6 +547,8 @@ func routeLabels(event *pb.TapEvent) string {
 	return out
 }
 
+// RenderTapEvent renders a Public API TapEvent to a string.
+// TODO: consider moving this into cli/cmd/tap.go.
```

Comment:
👍 Ah, right, this was originally moved here when we were rendering tap output as plaintext in the web UI, but we could move it back now that the web has its own rendering.

Reply:
Will do in a subsequent PR.